docs/content/_index.es.md

```diff
@@ -11,6 +11,13 @@ date: 2019-03-14T08:44:12+00:00
 
 En esta página web, está documentado Athens en detalle. Te enseñaremos que hace, por que es necesario, y como puedes ejecutarlo por tu cuenta. A continuación te mostramos un breve listado.
 
+#### ¿Cómo empezar?
+    Correr `docker run -p '3000:3000' gomods/athens:latest`
```

Comment:
Correr effectively means run, but only in the context of sport/physical exercise. It's better to use "Ejecutar" here, meaning "execute".